django-verdant/rca/templates/rca/home_page_2018.html

```diff
@@ -5,31 +5,54 @@
     <link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css"/>
 {% endblock %}
 
+{% block body_class %}{{ block.super }} template-home-page{% endblock %}
+
+{% block hero %}
+    {% image self.hero_image original as hero_image %}
```

Comment:
sorry, one extra comment: is it possible to not use the original here and to use a fill? It'll still be large but might be slightly less huge from a performance perspective?